code/modules/organs/subtypes/vaurca.dm

```diff
@@ -68,28 +68,187 @@
 
 /obj/item/organ/internal/vaurca/neuralsocket/removed(var/mob/living/carbon/human/target)
 	if(all_languages[LANGUAGE_VAURCA] in target.languages)
 		target.remove_language(LANGUAGE_VAURCA)
 		to_chat(target, "<span class='warning'>Your mind suddenly grows dark as the unity of the Hive is torn from you.</span>")
+	remove_verb(owner, all_hive_verbs)
+	..()
+
+/obj/item/organ/internal/vaurca/neuralsocket/admin
+	name = "administrative neural socket"
+	icon_state = "admin_socket"
+	desc = "The single most important organ for a Vaurca, able to copy their mind into their Virtual Reality Afterlife upon death. \
+	This one appears to be the far rarer administrative model including encrypted Hivenet access codes and an Emergency Remote Cast System. These are almost never found on any Vaurca Bioform except the Ta."
+	var/remote_cast = TRUE //get out of death free card
+	var/list/shielded_sockets = list() //sockets that you are currently protecting
+	var/list/shielded_mobs = list() //mobs that you are currently protecting
+	granted_verbs = list(
+		/mob/living/carbon/human/proc/hiveban,
+		/mob/living/carbon/human/proc/hivenet_neuralshock,
+		/mob/living/carbon/human/proc/hivevoid,
+		/mob/living/carbon/human/proc/hivenet_transmit,
+		/mob/living/carbon/human/proc/hivemute,
+		/mob/living/carbon/human/proc/hivenet_recieve,
+		/mob/living/carbon/human/proc/hivenet_encrypt,
+		/mob/living/carbon/human/proc/hivenet_camera,
+		/mob/living/carbon/human/proc/hivenet_lattice,
+		/mob/living/carbon/human/proc/hivenet_decrypt
+	)
+
+/obj/item/organ/internal/vaurca/neuralsocket/admin/process()
+	if(!owner)
+		return
+	if (is_broken())
+		if (all_languages[LANGUAGE_VAURCA] in owner.languages)
+			owner.remove_language(LANGUAGE_VAURCA)
+			to_chat(owner, SPAN_WARNING("Your mind suddenly grows dark as the unity of the Hive is torn from you."))
+	else
+		if (!(all_languages[LANGUAGE_VAURCA] in owner.languages) && !banned)
+			owner.add_language(LANGUAGE_VAURCA)
+			to_chat(owner, SPAN_NOTICE("Your mind expands, and your thoughts join the unity of the Hivenet."))
+
+	if(owner.stat == DEAD)
+		if(!remote_cast)
+			owner.visible_message(SPAN_WARNING("[owner]'s corpse spasms suddenly, legs twitching and antennae moving wildly as something hums beneath [owner.get_pronoun("his")] exoskeleton. After a moment, [owner.get_pronoun("he")] goes still."))
+			icon_state = "admin_socket_on"
+			desc += "The Emergency Remote Cast light is green, indicating it has been triggered."
+			remote_cast = 1
+		for(var/obj/item/organ/internal/vaurca/neuralsocket/S in shielded_sockets)
+			S.shielded = SOCKET_UNSHIELDED
+			to_chat(S.owner, SPAN_WARNING("You feel [owner]'s protection vanish from you, leaving your neural socket exposed."))
+			LAZYREMOVE(shielded_sockets, S)
+			LAZYREMOVE(shielded_mobs, S.owner)
+	..()
+
+/obj/item/organ/internal/augment/hiveshield
+	name = BP_HIVENET_SHIELD
+	organ_tag = BP_HIVENET_SHIELD
+	parent_organ = BP_HEAD
+	icon_state = "augment-pda" //placeholder
```

Comment:
```suggestion
	icon_state = "augment-pda"
```
if it's still a placeholder the comment isn't needed, unless there's plans in the near future to implement a replacement.